### PR TITLE
Test supported versions

### PR DESCRIPTION
**Pull request by arnetheduck**
Per readme, NimYAML supports Nim 1.4.0+ - this CI change keeps the readme honest.

### CODE REVIEW

File: .github/workflows/action.yml [hunk 1]
```diff
@@ -12,6 +12,8 @@ jobs:
           - windows-latest
           - macOS-latest
         nim-version:
+          - '1.4.x'
+          - '1.6.x'
```
Comment (arnetheduck) [inline .github/workflows/action.yml:16]:
```suggestion
          - '1.6.x'
```

**Comment by arnetheduck** [comment]
Looks like 1.6 is also failing - we use the library successfully with 1.6 (we would probably simply stick with this version if 1.6 support is dropped since 2.0 has quite a few problems still), so maybe it's a test-only problem.

**Comment by flyx** [comment]
The current HEAD has changes that use Nim 2.0 features, and thus doesn't work with Nim 1.6 anymore. I have yet to finish the release so that everything is properly documented.

If you stick to Nim 1.6, stay at the previous NimYAML release. There are only minor bugfixes in the new release and if you didn't encounter those bugs yet they probably don't affect you. In the case that you *do* have problems, please raise an issue since I will only continue maintaining the NimYAML 1.x line if I know that there are actually projects that benefit from it.

**Comment by arnetheduck** [comment]
> The current HEAD has changes that use Nim 2.0 features, and thus doesn't work with Nim 1.6 anymore. 

Alright, I'll close this PR for now (we use the 1.1 release indeed) - that said, I think it's useful to do something like this to HEAD as well so that version support regressions are caught during development - ie test explicit releases rather than tags such as stable which change over time - we don't upgrade yaml a lot, but when we do this would be helpful.

**Comment by flyx** [comment]
I incorporated this in the newest release by adding Nim `2.0.x` to the test matrix.

**Pull request closed without merging**